lib/data-export.ts

```
@@ -0,0 +1,340 @@
+function getDocumentDescription(doc: any): string {
+  const content = doc.content || {};
+  
+  switch (doc.type) {
+    case "resume":
+      return content.resumeData?.personal_info?.name || 
+             content.resumeData?.personalInfo?.name || 
+             content.personal_info?.name || 
+             content.personalInfo?.name || 
+             "Resume";
+    case "generated":
+      const sections = content.metadata?.sections || content.sections || [];
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)